templates/head.hbs

```diff
@@ -37,4 +37,9 @@
   outline: thin #c45e0a solid;
 }
 
+@media print {
+  .no-print, .no-print * {
```

Comment:
Do `.no-print` and `.no-print *` do something different in this case?  I would have thought `.-no-print` alone would cover descendants too.

Reply:
@davidlehn do not currently have cycles to deal with this. I believe the `*` is there to cover cases where descendants of an element get outside of the normal dom flow such as with a z-index or static position. The important also kind of worries me, but I believe I have seen things go to `display: none` with descendants go to `display: block` and the important would cover that case. This PR is not urgent.

Adding a To Do:

- [ ] @aljones15 needs to test what happens on hover with permutations of `display` prop in descendants with z-indexes outside the normal doc flow.